feelpp/feel/feelfilters/loadgmshmesh.hpp

```diff
@@ -140,15 +140,14 @@ BOOST_PARAMETER_FUNCTION(
         if ( physical_are_elementary_regions )
             import.setElementRegionAsPhysicalRegion( physical_are_elementary_regions );
         import.setRespectPartition( respect_partition );
-
         if ( rebuild_partitions && partitions > 1 )
         {
             _mesh_ptrtype _meshSeq = std::make_shared<_mesh_type>( Environment::worldCommSeqPtr() );
             _meshSeq->accept( import );
             _meshSeq->components().reset();
             _meshSeq->components().set( size_type(MESH_UPDATE_ELEMENTS_ADJACENCY|MESH_NO_UPDATE_MEASURES|MESH_GEOMAP_NOT_CACHED) );
             _meshSeq->updateForUse();
-
+#if defined(FEELPP_HAS_HDF5)
```

Comment:
I think is not a good place to put the hdf5 guard. It would be probably put in the PartitionIO class

Reply:
Sure, but there is also an unguarded `loadHDF5` below.

Reply:
this one OK

Reply:
BTW, PartitionIO is guarded by FEELPP_HAS_HDF5, so what are you suggesting exactly?

Reply:
sorry, forget, it's more delicate than I thought.